### ISSUE TITLE
[BUG] issue with Get-PnpTenant storage values.

### ISSUE THREAD

**Issue by shitalkhaire** [pnp/powershell#4557]
### Reporting an Issue or Missing Feature
I have Sites.FullControl.All permission still i do not getting correct Storagequota and even not getting allocated storage data.

### Expected behavior 
I want total storage of Sharepoint Tenant and used storage.
### Actual behavior 
i am getting Allocated storage is zero and  total Storage of Sharepoint admin center is different.

![image](https://github.com/user-attachments/assets/1a9460db-b245-45d5-93f0-910b094ad32b)

![image](https://github.com/user-attachments/assets/11ff5132-77de-4da0-ba0c-d3a9a464d525)

### Steps to reproduce behavior
$Url = "https://xyz-admin.sharepoint.com"
$ClientId = "123456678-45-576575"
Connect-PnPOnline -Url $AdminCenterURL -Interactive -ClientId $CleintId 
$tenantInfo = Get-PnPTenant
$Storagequota = $tenantInfo.StorageQuota
$Allocatedquota = $tenantInfo.StorageQuotaAllocated

### What is the version of the Cmdlet module you are running?
 PnP.PowerShell 2.12.0   

### Which operating system/environment are you running PnP PowerShell on?
- [ ] Windows

**Comment by jackpoz** [comment]
Storage quota is the same as on SPO Admin Center, as it shows 1355776 MB, which is ~1,293 TB.

What do you get with [Get-SPOTenant](https://learn.microsoft.com/en-us/powershell/module/sharepoint-online/get-spotenant?view=sharepoint-ps) ? In my case I get the same numbers:

![image](https://github.com/user-attachments/assets/f2fe1e85-f7f8-4fc0-af26-30d50d671374)

![image](https://github.com/user-attachments/assets/4e4b0398-ee60-40bf-8ead-ae08c81731b1)

Looks more like an issue in SPO api itself than PnP PowerShell.

**Comment by shitalkhaire** [comment]
Yes,I am getting same numbers in storageQuota with both cmdlets:
1) PNP:
![image](https://github.com/user-attachments/assets/58d6432e-748e-42b2-a1a1-4c9191f7aa28)

2) SPO:
![image](https://github.com/user-attachments/assets/0a5c68c0-c648-4f0a-ab98-e975659ce2f4)

**Comment by jackpoz** [comment]
According to https://techcommunity.microsoft.com/discussions/sharepoint_general/what-is-the-storagequotaallocated-means-in-spo/297042 , the StorageQuotaAllocated is not the amount of space used, which is why it's 0.

**Comment by jackpoz** [comment]
I checked my scripts and found 2 ways:
- [Get-SPOGeoStorageQuota](https://learn.microsoft.com/en-us/powershell/module/sharepoint-online/get-spogeostoragequota?view=sharepoint-ps) returns GeoUsedStorageMB as used capacity, matching what SPO Admin Center shows. It returns also TenantStorageMB as total:
![image](https://github.com/user-attachments/assets/e04e683e-c2ff-4900-944e-8a1769e48e00)
- `(Invoke-PnPSPRestMethod -Url "/_api/StorageQuotas()?api-version=1.3.2").value` returns the same
![image](https://github.com/user-attachments/assets/036df38f-b589-4c73-b9cc-27e9f7848f01)

**Comment by gautamdsheth** [comment]
Looks like there's a workaround for this and also for the original issue MS needs to be involved. Since we wont be able to do much here, am closing this.
